_posts/en/2016-xx-xx-measure-operation-time-using-console-object.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+
+title: Measure Operation time using console object
+tip-number: xx
+tip-username: SarjuHansaliya
+tip-username-profile: https://github.com/SarjuHansaliya
+tip-tldr: Measure Operation time using `console` object
+
+categories:
+    - en
+---
+
+Sometime we need to measure time of some tasks to analyze how much time it takes to execute. To achieve that, normal practice is to note down time before starting of the task and note down time after completion of the task, and then take a difference between two times.But this is a lot of work for only time measuring.
```

Comment:
"starting the task" and "completing the task" is better.
"take the difference" 
Space after dot missing. (Before "But")